src/runtime_src/core/pcie/windows/shim.cpp

```diff
@@ -903,7 +903,7 @@ struct shim
     if (!status || bytes != sizeof(struct mem_topology))
       throw std::runtime_error("DeviceIoControl IOCTL_XOCL_STAT (get_mem_topology) failed");
 
-    DWORD mem_topology_size = sizeof(struct mem_topology) + (mem_info.m_count - 1) * sizeof(struct mem_data);
+    DWORD mem_topology_size = sizeof(struct mem_topology) + (mem_info.m_count) * sizeof(struct mem_data);
```

Comment:
I think the point is that sizeof(mem_topology) already contains the size of one mem_data.  This is indeed an implicit assumption that if true should be mentioned in a comment prior to the original line here.

I believe same is true for the other xclbin data types you size up below.

Reply:
True, fixed this, thank you.